frontend/packages/policy-editor/src/components/SecurityLevel/SecurityLevel.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode } from 'react';
+import classes from './SecurityLevel.module.css';
+import { Select, Label, Paragraph, HelpText, Link } from '@digdir/design-system-react';
+import { useTranslation, Trans } from 'react-i18next';
+import { RequiredAuthLevel } from '../../types';
+
+const SELECT_AUTH_LEVEL_ID: string = 'select-auth-level';
+const URL_TO_SECURITY_LEVEL_PAGE: string =
+  'https://info.altinn.no/hjelp/innlogging/diverse-om-innlogging/hva-er-sikkerhetsniva/';
+
+export const authlevelOptions = [
+  { value: '0', label: 'policy_editor.auth_level_option_0' },
+  { value: '1', label: 'policy_editor.auth_level_option_1' },
+  { value: '2', label: 'policy_editor.auth_level_option_2' },
+  { value: '3', label: 'policy_editor.auth_level_option_3' },
+  { value: '4', label: 'policy_editor.auth_level_option_4' },
+];
+
+export type SecurityLevelProps = {
+  requiredAuthenticationLevelEndUser: RequiredAuthLevel;
+  onSave: (authLevel: RequiredAuthLevel) => void;
+};
+
+export const SecurityLevel = ({
```

Comment:
Ha du vurdert å bruke et mer beskrivende navn for denne, f.eks. `SecurityLevelSelect`, for å gjøre det klart at det er en komponent hvor brukeren skal fylle ut noe?